src/views/Vault/components/DepositAssetRow.tsx

```diff
@@ -76,7 +75,11 @@ const DepositAssetRow: React.FC<DepositAssetRowProps> = ({
 					</Row>
 				</Col>
 				<Col xs={11} sm={6} md={6} className="balance">
-					<Value value={balanceUSD} numberPrefix="$" decimals={2} />
+					<Value
+						value={balanceUSD === 'NaN' ? '0.00' : balanceUSD}
```

Comment:
Checking strict equality on NaN will not work like this.
You should use [`isNaN`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN).